python/gym_ignition/rbd/utils.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (C) 2020 Istituto Italiano di Tecnologia (IIT). All rights reserved.
+# This software may be modified and distributed under the terms of the
+# GNU Lesser General Public License v2.1 or any later version.
+
+import numpy as np
+
+
+def hat(vector3: np.ndarray) -> np.ndarray:
```

Comment:
Not a big problem, but the symbol that in latex is typically used for representing this operator is the wedge, not the hat.

Reply:
Yeah, makes sense to match latex terminology here :+1:

Reply:
I start having second thoughts on this, using _hat_ instead of _wedge_ would give the code a more colloquial sense since standing to the notation document the operator is pronounced hat while in latex is typesetted as wedge.

Reply:
> the operator is pronounced hat while in latex is typesetted as wedge.

I guess there is no absolute reference, but for me $\hat{a}$ is "hat" I.e. the hat is on top of the symbol, while $a^\wedge$ is wedge (i.e. where the symbol is on the top right, like an exponential.